src/test/java/fi/metatavu/metamind/bot/slots/LocalizedNumberSlotTest.java

```diff
@@ -0,0 +1,30 @@
+package fi.metatavu.metamind.bot.slots;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.io.IOException;
+
+import org.junit.Test;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.rabidgremlin.mutters.core.SlotMatch;
+
+import fi.metatavu.metamind.slot.models.NumberSlotTestModels;
+
+public class LocalizedNumberSlotTest {
+
+  @Test
+  public void testFinnishWordStringToNumber() throws IOException {
+    ObjectMapper objectMapper = new ObjectMapper();
+    NumberSlotTestModels testModels = objectMapper.readValue(getClass().getClassLoader().getResourceAsStream("fi/metatavu/metamind/bot/slots/number-slot-tests.json"), NumberSlotTestModels.class);
+    LocalizedNumberSlot localizedNumberSlot = new LocalizedNumberSlot("test");
+    
+    testModels.getFi().forEach((testModel) -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the parentheses around the "testModel" parameter [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1611)